tests/cores/telemetry/test_telemetry.py

```diff
@@ -6,7 +6,7 @@
 import time
 import unittest
 
-os.environ["ENABLE_OPEA_TELEMETRY"] = "True"
+os.environ["TELEMETRY_ENDPOINT"] = "http://jaeger:4318/v1/traces"
```

Comment:
I see this is already merged, but I don't think its best practice to hardcode a hostname in the python.
I would revert this. CI config should live in separate config files.

Reply:
Is that test intended only for Kubernetes (where DNS maps names to services)?   Then above endpoint would be port of a "jaeger" service, not a host called "jaeger".

Reply:
Excellent points @eero-t and @dcmiddle.

Reply:
yes. jaeger is a service name given during gaudi-default or xeon-default network. the service name is assigned as docker instance name in docker compose